test/library/packages/LinearAlgebra/performance/diag/setdiag-test.chpl

```diff
@@ -2,7 +2,7 @@ use LinearAlgebra;
 
 config const m=10;
 
-const Space = {1..m, 1..m};
+const Space = {42..#m, 24..#m};
```

Comment:
Can you keep the old case and this as a separate case? Note that it doesn't need to be in a separate test. Just expand this test to check:

- 1-based domains
- 0-based domains
- negative-based domains
- something random like what you have here